core/src/test/java/hudson/slaves/ComputerLauncherTest.java

```diff
@@ -36,15 +36,15 @@
 
 public class ComputerLauncherTest {
 
-    @Test public void jdk7() throws IOException {
+    @Test(expected=IOException.class) public void jdk7() throws IOException {
```

Comment:
Exception on a separate line would be preferable

Reply:
Just followed existing style in this file.